src/vl-upload.js

```diff
@@ -384,4 +402,23 @@ export class VlUpload extends vlFormValidationElement(vlElement(HTMLElement)) {
   _enableAutoProcessQueue() {
     this._dropzone.options.autoProcessQueue = true;
   }
+
+  _addSlots() {
+    Promise.all([this.uploadElement]).then(() => {
+      const subTitleSlotNodes = this._subTitleSlotElement.assignedNodes();
+      if (subTitleSlotNodes.length > 0) {
+        const small = this.uploadElement.querySelector('small');
+        small.innerHTML = subTitleSlotNodes[0].outerHTML;
+      }
+
+      const titleSlotNodes = this._titleSlotElement.assignedNodes();
+      if (titleSlotNodes.length > 0) {
+        const buttonContainer = this.uploadElement.querySelector('.vl-upload__element__button__container');
+        buttonContainer.innerHTML = titleSlotNodes[0].outerHTML;
+      }
+    }).finally(() => {
+      this._titleSlotElement.remove();
+      this._subTitleSlotElement.remove();
```

Comment:
waarom de remove?

Reply:
beschouwde dat als een vorm van opkuis, had initieel het attribuut hidden toegevoegd maar dit leek me dan uiteindelijk properder

Reply:
Zijn de <slot> tags uberhaupt wel nodig? Een van de grote voordelen is dat de browser ervoor zorgt dat de html die door de gebruiker meegegeven wordt, overgenomen wordt in de slot tag, maar dat gebruiken we hier niet.
Volgens mij kan je die weglaten en dan `this.querySelector('[slot="title"]')` doen om de meegegeven slot te selecteren en daar de html van over te nemen. Dan moet je geen cleanup/hidden doen.

Reply:
gisteren getest en via die this.querySelector kan ik weldegelijk aan de bovenliggende slot.
Nu voorlopig nog niet toegepast @coemans wat denk jij hier van?

Nu wat ik nog af vraag als we de slot content overnemen gaat deze wrs niet getoond worden omdat we zelf geen slots meer zetten vermoed?

Reply:
De huidige code werkt eigenlijk niet met slots, ik ga een poging doen om de native slot werking toe te passen zodat er ook geen mutation observers nodig zijn om wijzigingen aan de slot elementen door te voeren.